### DISCUSSION TITLE
Collision Force

### DISCUSSION THREAD

**Discussion by Aystein** [cosmograph-org/cosmos#64] [Q&A]
Hey, thanks for creating this library. It looks really promising and is quite useful already for visualizing our graphs.
However for a second use case we would need just a fast force layout without links but with nodes only. Each node should be driven to a x/y position and they should have collision detection (bee swarm plots). The d3 equivalent would be forceX/forceY and forceCollision.

Are features like those planned at any point and are they possible? (I could imagine that the collision force is quite hard to do)

**Comment by rokotyan** [comment]
Thank you for your feedback and kind words about the library. I'm glad to hear that it has been useful for visualizing your graphs.

Regarding your second use case, developing an optimized collision detection algorithm for GPU can be quite challenging, and it might not provide the desired performance benefits (comparing the what we were able to achieve with the force layout). So we have no plans to work on something like that in the foreseeable future. I won't say never, though.
